### ISSUE TITLE
About  magic_point_coco_train.yaml 

### ISSUE THREAD

**Issue by foxkw** [shaofengzeng/SuperPoint-Pytorch#22]
Hi, thanks for your great job.
I want to train magicpoint on kitti , so is it needed to use magic_point_coco_train.yaml ? I noticed there is a path of labels.  but how to ensure its correctness so as to train magicpoint ? because  if the label result generated in the previous step is average, what is the significance of this step of train magicpoint ?

**Comment by shaofengzeng** [comment]
Sorry for the late reply. I think you have to use magic_point_coco_train.yaml to train your magicpoint, the training process is independent of data. magic_point_coco_train.yaml is used to train the magicpoint model with labeled coco images (the labels are generated by magic model trained by synthetic data, you can learn more by reading the readme.txt). In general, visualization is the simplest and most effective way to verify your training results, so just show your predictions~

**Comment by foxkw** [comment]
> Sorry for the late reply. I think you have to use magic_point_coco_train.yaml to train your magicpoint, the training process is independent of the data. magic_point_coco_train.yaml is used to train the magicpoint model with labeled coco images (the labels are generated by magic model trained by synthetic data, you can learn more by reading the readme.txt). In general, visualization is the simplest and most effective way to verify your training results, so just show your predictions~

I found that the magipoint model will miss some points when export the label on images, so that the corner detection of the lane is unstable in the continuous frame image. This seems to indicate that magipoint also has shortcomings in the export label stage. Do you have any good ideas, improve it, and look forward to your reply

**Comment by shaofengzeng** [comment]
We select feature points according to the probability of predicted by magicpoint (as well as NMS), therefore, some points that people think should be feature points may be omitted

**Comment by ps-star** [comment]
When using the coco dataset to train magipoint, the coco2017 training set label belongs to the json file in the annotations folder, not the. npy file required for training. How to solve this problem
